### PR TITLE
add html-css-popup

### PR DESCRIPTION
**Pull request by alstep07**
### hooli-style-popup

[Demo](https://alstep07.github.io/) |
[Code base](https://github.com/alstep07/kottans-frontend/tree/master/task_hooli_style_popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/alstep07/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8" />
```
Comment (lenadgit) [inline submissions/alstep07/html-css-popup/index.html:5]: The single tags should be without slashes "/", could you please fix it for all single tags?

**Review by lenadgit — CHANGES_REQUESTED** [review]

File: submissions/alstep07/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+			<li class="nav__elem">
+				<a class="nav__btn" href="https://github.com/alstep07/kottans-frontend">
+					<img class="user__img" src="./img/user_photo.png" alt="user_photo">
+				</a>
+			</li>
+		</ul>
+	</nav>
+</body>
+
+</html>
```
Comment (lenadgit) [inline submissions/alstep07/html-css-popup/index.html:127]: could you please add an empty line at the end of the file?  if you see this symbol - it means that the empty line is needed   
empty line

File: submissions/alstep07/html-css-popup/style.css [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+    background-color: #ffb694;
+}
+
+.menu__extra-input:checked~.menu__extra {
+    display: block;
+}
+
+.menu__extra-input:checked+.menu__extra-btn {
+    display: none;
+}
```
Comment (lenadgit) [inline submissions/alstep07/html-css-popup/style.css:195]: could you please add an empty line

**Review by lenadgit — CHANGES_REQUESTED** [review]

**Review by A-Ostrovnyy — APPROVED** [review]
Good work!

**Comment by A-Ostrovnyy** [comment]
@lenadgit your turn)

File: submissions/alstep07/html-css-popup/index.html (hunk 1 above)
Comment (alstep07) [reply submissions/alstep07/html-css-popup/index.html]: fixed

File: submissions/alstep07/html-css-popup/index.html (hunk 2 above)
Comment (alstep07) [reply submissions/alstep07/html-css-popup/index.html]: added

File: submissions/alstep07/html-css-popup/style.css (hunk 3 above)
Comment (alstep07) [reply submissions/alstep07/html-css-popup/style.css]: added

**Comment by alstep07** [comment]
Please, review my solution.

**Review by lenadgit — APPROVED** [review]

**Pull request merged**